docs/js/react.js

```diff
@@ -10,7 +10,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * 
+ *
```

Comment:
I got really confused there for a sec, thank's for clarifying that - I'll amend other files in a minute